Vagrantfile

```diff
@@ -152,6 +152,15 @@ fi
 SCRIPT
 
 
+INSTALL_PYTHON_ON_UBUNTU = <<-SCRIPT
+#!/bin/bash
+
+apt install python -y
+
+SCRIPT
```

Comment:
You should write a proper commit message that explain why we need to install Python 2.7 manually (and also python-apt).

Reply:
Explanation done in the last commit (requirement for the salt-minion)